shared/css/phase1-design-system.scss

```diff
@@ -76,6 +76,13 @@ a {
 
 p {
   color: $neutral_dark;
+
+  &.overline {
```

Comment:
@levadadenys Can you okay this addition to the phase1-design-system sheet?

Reply:
(Side note: for the Phase II updates I'm planning on making an importable `typography.scss` file that will include shared styles like this one 🙌 )

Reply:
@kelbyhawn I thought we agreed to separate styles for dashboard and pegasus. This component is meant to be used at dashboard, so not sure we need to use pegasus styles here. Also, there're typography components coming soon

Reply:
@levadadenys we are going to separate them, but we talked about doing a shared `typography.scss` file that we can import into the respective dashboard and pegasus rebrand scss files since those should be the same — we talked about it [here](https://codedotorg.slack.com/archives/C04EDUXCZSN/p1678388583725209) w/ Mark.

Reply:
I went ahead and moved these into the local stylesheet until typography is available